app/experimenter/docs/adrs/0001-create-new-nimbus-application.md

```diff
@@ -0,0 +1,119 @@
+# Create A New Web Application To Manage Nimbus Experiments
+
+* Status: proposed
+* Deciders: Kate Hudson, Jared Lockhart, TBD
+* Date: 2020-08-18
+
+## Context and Problem Statement
+
+Experimenter is a web application developed solely for the purpose of managing product experiments at Mozilla.  It was originally designed to manage the Shield experimentation workflow, and has expanded to include a comprehensive workflow for managing Normandy experiments.  Project Nimbus is being developed with very different core assumptions than the previous experimentation platforms, and it is unclear whether building on top of the existing Experimenter infrastructure or creating a new purpose specific application will best suit the project's needs.
+
+## Decision Drivers <!-- optional -->
+
+* Meeting Project Nimbus' needs in a rapid, continuous fashion with minimal downtime
+* Contributability by a large, distributed group of people with different backgrounds (web engineers, data engineers, data scientists)
+* Fitting cohesively into the FXA engineering pipeline/culture
+* Continuing to support the existing experimentation (Normandy) workflow
+
+## Considered Options
+
+* Build On Top Of Experimenter
+* Start A New Application In Python/Django
+* Start A New Application in TypeScript/Node
+
+## Decision Outcome
+
+Chosen option: TBD
+
+### Positive Consequences
+
+* [e.g., improvement of quality attribute satisfaction, follow-up decisions required, …]
+* …
+
+### Negative Consequences
+
+* [e.g., compromising quality attribute, follow-up decisions required, …]
+* …
+
+## Pros and Cons of the Options
+
+### Build On Top Of Experimenter
+
+Much of the existing Experimenter infrastructure can be reused, which gives Project Nimbus a well established framework to build on top of.
+
+- Pros
+  - Single source of truth for all past/future experiments
```

Comment:
I would also get @shell1 's feedback on this since she has been working closely with leadership on this

Reply:
It is maximally critical to get be able to utilize a single source of truth for experimentation.  Not just between Normandy and Nimbus, but with the various web platforms, and mobile.  

Stakeholders are leadership, anyone planning experiments, and anyone needing to know what is going on in a product area (ex: PM).  The goal is to easily be able to learn, inform current plans, and present our learnings as part of the direction narrative for a product area.  For that we need to be able to answer:

"what hypothesis have we tested in the past, are we testing currently, or will we test in the future for ___ area".  "how did we measure those hypothesis?" 

One big weakness is different teams experimenting in the same area - in different ways (ex: Product and Marketing running experiments in same area - but results not cross-shared.  

We want to increase our ability to weave a complete narrative.

Reply:
I was hoping that integrating experiment reporting into Experimenter would allow us to use Experimenter as a results repository. Stakeholders do seem to want one-stop shopping, and Jetstream and the reporting interface will support both Nimbus- and Normandy-flavored experiments. If we end up with a legacy Normandy-flavored product and a new Nimbus-flavored product and we end up supporting both for 6-12 months, that story gets a little harder to write.